### PR TITLE
Fix recent unit tests fail on windows #970

### LINKED ISSUE #970: Recent unit tests fail on Windows
Reasons for failure are related to 

- EOL difference between Linux (\n) and Windows (\r\n)
- Separator (/ in Linux and \ in Windows)

### PR DESCRIPTION
**Pull request by aalhossary**
Fixes #970

### CODE REVIEW

**Comment by josemduarte** [comment]
Thanks @aalhossary 

@richarda23 could you have a look at this? I think this touche some of the tests you recently wrote.

**Comment by richarda23** [comment]
Sure will take a look in the next 2 days
  
  
   
    On 27/09/2021 21:42 Jose Manuel Duarte ***@***.***> wrote:
   
   
    
   
   
    
   
   
   Thanks @aalhossary
   @richarda23 could you have a look at this? I think this touche some of the tests you recently wrote.

**Review by richarda23 — COMMENTED** [review]
checked out and tested that still pass on linux.

File: biojava-core/src/test/java/org/biojava/nbio/core/util/PrettyXMLWriterTest.java [hunk 1]
```diff
@@ -68,7 +68,8 @@ void specialCharsAreEscaped() throws IOException {
         xw.print("<code>some literal xml </code>");
         xw.closeTag("dt");
 
-        assertEquals(EXPECTED, sw.toString());
+        if(! StringManipulationHelper.equalsToIgnoreEndline(EXPECTED, sw.toString()))
```
Comment (richarda23) [inline biojava-core/src/test/java/org/biojava/nbio/core/util/PrettyXMLWriterTest.java:71]: Using asserts is more idiomatic (but no more correct). e.g. assertTrue(expected, actual, messageOnFail), here and elsewhere in the file
```
 assertTrue(StringManipulationHelper.equalsToIgnoreEndline(EXPECTED, sw.toString()),
        String.format("Strings are not equal (ignoring endline differences. expected: [%s], but it was:[%s]",
         EXPECTED, sw.toString()));
```

Comment (aalhossary) [reply biojava-core/src/test/java/org/biojava/nbio/core/util/PrettyXMLWriterTest.java:71]: OK. 👍

**Review by josemduarte — APPROVED** [review]
LGTM thanks

**Pull request merged**